package.json

```diff
@@ -8,7 +8,8 @@
     "Daniel Schafer <dschafer@fb.com>"
   ],
   "license": "BSD-3-Clause",
-  "main": "index.js",
+  "main": "es5/index.js",
+  "jsnext:main": "modules/index.js",
```

Comment:
@baer, `jsnext:main` [is deprecated](https://github.com/rollup/rollup/issues/969). Use `module` instead.